pages/services/prometheus/0.1.0-2.3.1/deploy-best-practice/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: layout.pug
+navigationTitle:  Deployment Best Practices
+title: Deployment Best Practices
+menuWeight: 30
+excerpt: Best practices for production deployment
+featureMaturity:
+enterprise: false
+---
+# Deployment Best Practices
+
+Note that the Node Exporter is best run on the host system, uncontainerized, so that it can cleanly expose all the host system metrics.
+
+## Configuration Best Practices for Production
+## Hardware Sizing Recommendations
+
+In practice this can only really be determined empirically, as it varies organisation by organisation and machine by machine.
```

Comment:
do not add links to blog posts. COnfiguration best practices must come from design doc.

Reply:
Removed blog post